core/src/test/scala/akka/kafka/scaladsl/MultiConsumerSpec.scala

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (C) 2014 - 2016 Softwaremill <http://softwaremill.com>
+ * Copyright (C) 2016 - 2018 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.kafka.scaladsl
+
+import akka.Done
+import akka.kafka.test.Utils._
+import net.manub.embeddedkafka.EmbeddedKafkaConfig
+
+import scala.collection.immutable
+import scala.concurrent.duration._
+import scala.concurrent.{Await, Future}
+
+class MultiConsumerSpec extends SpecBase(kafkaPort = 9032) {
+
+  def createKafkaConfig: EmbeddedKafkaConfig =
+    EmbeddedKafkaConfig(
+      kafkaPort, zooKeeperPort,
+      Map(
+        "broker.id" -> "1",
+        "num.partitions" -> "3",
+        "offsets.topic.replication.factor" -> "1",
+        "offsets.topic.num.partitions" -> "3"
+      ))
+
+  "Multiple consumer in a single consumer group" must {
+
+    "together read all data from multiple topics" in assertAllStagesStopped {
+      val topics = List(
+        createTopic(0),
+        createTopic(1),
+        createTopic(2))
+      val group1 = createGroup(1)
+      val group2 = createGroup(2)
+
+      // produce 10 batches of 10 elements to all topics
+      val batches = 10
+      val batchSize = 10
+      Await.result(produceBatches(topics, batches, batchSize), remainingOrDefault)
+
+      val consumerSettings = consumerDefaults.withGroupId(group1)
+
+      val (control1, probe1) = createProbe(consumerSettings, topics: _*)
+      val (control2, probe2) = createProbe(consumerSettings, topics: _*)
```

Comment:
Shouldn't this use `group2`?

Reply:
Ahh, this test is for multiple consumers in a single group. So it is fine here. But line `36` then should be removed?